Print_A_Bot/general/utils_view.py

```diff
@@ -0,0 +1,61 @@
+import json
+import logging
+
+from django.conf import settings
+from django.contrib import messages
+from django.http import HttpResponse
+from django.shortcuts import render
+
+
+logger = logging.getLogger(__name__)
+
+
+def return_json(data, **kwargs):
```

Comment:
these pre-date django rest framework and are still in use by a couple of the views above -- porting them to use `@api_view` would be a worthwhile effort